packages/renderer/src/lib/images/LoaderAnimation.svelte

```diff
@@ -0,0 +1,136 @@
+<script lang="ts">
+import { onMount } from 'svelte';
+
+export let size = 400;
+
+let dotsGroup: SVGGElement | null = null;
+let eyesElement: SVGUseElement | null = null;
```

Comment:
issue: null should not be used as per https://github.com/kortex-hub/kortex/blob/main/eslint.config.mjs#L30 rule